src/rf/apps/core/models.py

```diff
@@ -133,6 +133,10 @@ class Meta:
     thumb_large_key = CharField(max_length=255, blank=True, default='',
                                 help_text='S3 key for large thumbnail')
 
+    def has_copied_images(self):
+        image = self.layer_images.first()
```

Comment:
I actually think it has a pretty obvious use case:
 * You have a high res geotiff you took but want to fill in some missing data with something like landsat or other open imagery.